src/artm/messages.proto

```diff
@@ -387,6 +389,21 @@ message TopicKernelScore {
   optional StringArray topic_name = 10;
 }
 
+// Represents a configuration of a topic part in nwt score
+message TopicPartInNwtScoreConfig {
+  optional float eps = 1 [default = 1e-37];
+  optional string class_id = 2 [default = "@default_class"];
+  repeated string topic_name = 3;
+  repeated string topic_name_to_score = 4;
```

Comment:
@MelLain , do you think ``TopicMassPhi`` would be a good name? I think @anyap was ok with it, so if you too agree then let's conclude on it :)